docker/scripts/install_python_version

```diff
@@ -43,8 +44,7 @@ for python_version in "${array[@]}"; do
   rm -rf /var/cache/apt/*
 
   echo "###### Installing pip for Python ${python_version} ######"
-  python${python_version} -m easy_install pip
-  #python${python_version} -m pip -U install pip
+  python${python_version} -m easy_install 'pip<21'
```

Comment:
I would only do this for Python 2.x, for the Python 3 we want to be moving forward

Reply:
good call